setup.py

```diff
@@ -42,7 +42,7 @@
     "google-api-core[grpc] >= 1.34.0, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*",
     "proto-plus >= 1.22.0, <2.0.0dev",
     "proto-plus >= 1.22.2, <2.0.0dev; python_version>='3.11'",
-    "protobuf>=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
+    "protobuf>=3.19.5,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
```

Comment:
Please add an upper bound `<6.0.0` so that users aren't broken when a new major version of protobuf is released. Also , bump the minimum version to `protobuf>=3.20.2` which matches what we have in `python-api-common-protos`

https://github.com/googleapis/python-api-common-protos/blob/c77c0dc5d29ef780d781a3c5d757736a9ed09674/setup.py#L26-L28

```suggestion
    "protobuf>=3.20.2,<6.0.0dev,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
```

Otherwise LGTM